### DISCUSSION TITLE
Disconnecting constantly

### DISCUSSION THREAD

**Discussion by Hainz7** [mmpneo/simple-obs-stt#1] [Q&A] [answered]
Any idea why it keeps disconnecting and I have to minimise what I am doing to reconnect it?

**Comment by mmpneo** [comment, marked as answer]
@Hainz7 hey
- what part of the app is disconnecting? (stt, tts, obs?)
- desktop or web version, if web -> which browser

https://discord.gg/SMKjA2yGf7 also if you have discord, we can move there

**Reply by Hainz7** [reply]
Using the Web version on Chrome and the stt keeps disconnecting also the stt is set to Native, English, United States
![image](https://user-images.githubusercontent.com/81328227/124761794-13476a80-df2a-11eb-9e8e-dd1407697fa6.png)

**Reply by mmpneo** [reply]
I just pushed small patch to the app, you should be able to see the disconnect reason now
![image](https://user-images.githubusercontent.com/3977499/124770224-d6d43880-df42-11eb-9bb4-72aa4a8aa31a.png)

**Reply by mmpneo** [reply]
I also found that chrome cannot handle multiple tabs with active microphone input, could it be the reason?

**Reply by Hainz7** [reply]
Microphone is not using multiple tabs and it still is disconnecting and says the reason is network like your picture above
